release-calendar/src/server/repository-service.ts

```diff
@@ -73,4 +73,16 @@ export class RepositoryService {
       }),
     );
   }
+
+  async createReleases(releases: Release[]): Promise<void> {
+    await this.releaseRepository.save(releases).catch((error) => {
+      throw error;
```

Comment:
If an `await`-ed Promise rejects, I believe it will throw synchronously. Unless I'm misinformed, this `catch` won't do anything

Reply:
It's a TypeORM quirk - https://github.com/typeorm/typeorm/issues/5057

Reply:
Gotcha, thanks for the reference! Maybe include a comment with this link?

Reply:
Okiedokes